pkg/database/database.go

```diff
@@ -42,31 +33,25 @@ func InitDb() {
 }
 
 func dbConfig() map[string]string {
-	conf := make(map[string]string)
-	host, ok := os.LookupEnv(dbhost)
-	if !ok {
-		panic("DBHOST environment variable required but not set")
-	}
-	port, ok := os.LookupEnv(dbport)
-	if !ok {
-		panic("DBPORT environment variable required but not set")
-	}
-	user, ok := os.LookupEnv(dbuser)
-	if !ok {
-		panic("DBUSER environment variable required but not set")
-	}
-	password, ok := os.LookupEnv(dbpass)
-	if !ok {
-		panic("DBPASS environment variable required but not set")
-	}
-	name, ok := os.LookupEnv(dbname)
-	if !ok {
-		panic("DBNAME environment variable required but not set")
+	viper.AddConfigPath("/Users/jordanlange/Documents/projects/personal/backlog-api/pkg/config")
+	viper.SetConfigName("config")
+	viper.SetConfigType("yaml")
+	err := viper.ReadInConfig()
+	if err != nil {
+		fmt.Println(err)
 	}
-	conf[dbhost] = host
-	conf[dbport] = port
-	conf[dbuser] = user
-	conf[dbpass] = password
-	conf[dbname] = name
+	
+	conf := make(map[string]string)
+	host := viper.GetString("database.dbhost")
+	port := viper.GetString("database.dbport")
+	user := viper.GetString("database.dbuser")
+	password := viper.GetString("database.dbpass")
+	name := viper.GetString("database.dbname")
+
+	conf["dbhost"] = host
+	conf["dbport"] = port
+	conf["dbuser"] = user
+	conf["dbpass"] = password
+	conf["dbname"] = name
```

Comment:
Instead of a map, maybe this can be refactored into a struct

Reply:
What's the benefit of a struct in this case?